### PR TITLE
Android text input fixes

### PR DESCRIPTION
**Pull request by EphemeralX**
* Fix for an issue on Android where the keyboard won't show up on UsernameSetupScreen and CreateRoomScreen. Add touchable space on the text input to focus/show keyboard.
* Fix for the background image shifting up on Android when the keyboard shows up or disappears.
* Fix for missing imports in ProfileDrawer

### CODE REVIEW

File: civilapp/src/browse/CreateRoomScreen.js [hunk 1]
```diff
@@ -104,19 +104,30 @@ export default class CreateRoomScreen extends React.Component {
...
+          <TextInput
+            autoCorrect={false}
+            keyboardAppearance="dark"
+            blurOnSubmit={false}
+            value={this.state.roomTitle}
+            onChangeText={text => this.setState({ roomTitle: text })}
+            maxLength={24}
+            selectionColor={'#D74BE0'}
+            autoFocus
+            style={{
```
Comment (rauchp) [inline civilapp/src/browse/CreateRoomScreen.js]: nit: can you put this in a style sheet?

File: civilapp/src/current/RoomDrawer.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import RoomDetails from './RoomDetails.js'
+import { withNavigation } from 'react-navigation'
+class TestDrawer extends React.Component {
```
Comment (rauchp) [inline civilapp/src/current/RoomDrawer.js]: nit: can you rename class to RoomDrawer?

Comment (EphemeralX) [reply civilapp/src/current/RoomDrawer.js]: whoa, I didn't even touch or see this, wtf

File: civilapp/src/browse/CreateRoomScreen.js (hunk 1 above)
Comment (EphemeralX) [reply civilapp/src/browse/CreateRoomScreen.js]: sure, calling it hiddenHitbox

**Pull request merged**